js/preload/preloadMain.js

```diff
@@ -209,7 +209,15 @@ function createAPI() {
          * see: https://electron.atom.io/docs/api/desktop-capturer/
          * for interface: see documentation in desktopCapturer/getSources.js
          */
-        getMediaSources: getMediaSources
+        getMediaSources: getMediaSources,
+
+        showAlertSettings: function() {
```

Comment:
Sorry, I am nitpicking a bit here. We still want this to be "showNotificationSettings". Consistency please :)